assets/src/components/activities/adaptive/components/authoring/LayoutEditor.tsx

```diff
@@ -401,7 +401,7 @@ const LayoutEditor: React.FC<LayoutEditorProps> = (props) => {
       }
     },
     ['Delete', 'Backspace'],
-    {},
+    { ctrlKey: true },
```

Comment:
changing the shortcut key for 'Delete' and 'Backspace' to `'Cntrl + Delete'` and ` 'Cntrl + Backspace' `